backend/src/Controller/EmailVerificationController.php

```
@@ -21,11 +22,22 @@ public function verify(
         string $token,
         UserRepository $userRepository,
         EntityManagerInterface $entityManager,
-        Request $request
+        Request $request,
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)